communication/slack-backup/README.md

```diff
@@ -0,0 +1,136 @@
+# HOWTO Backup Private Channels and DMs from Slack
+
+To back up private channels, and direct message chains, from Slack we currently recommend [slackdump](https://github.com/rusq/slackdump).
+
+It is a relatively easy to use commandline tool to backup your channels and DMs and store the data locally. No fees or 3rd-party online storage is involved, making it suitable for even confidential private channels.
+
+## Preparing to Export
+
+Decide which member of your private channel will archive it. This should be someone who is prepared to follow up before the June 20th deadline, and can easily install and use Slackdump. You should also give some thought as to [what you will do with the archive files](#what-should-i-do-with-the-backup-files) afterwards, although the critical thing is to make sure that someone makes a copy before the deadline.
+
+## Installing Slackdump
+
+Slackdump is supporting various platforms. You can find the latest release [here](https://github.com/rusq/slackdump/releases/tag/v3.1.4). The installation instructions below are taken from [the official instructions of the project](https://github.com/rusq/slackdump/?tab=readme-ov-file#installation-and-quickstart) with a few additions based on our experience.
+
+### MacOS
+
+The easiest way to install on a Mac is using Brew:
+
+```
+brew install slackdump
+```
+
+Some Linux distros also support Brew, which can be an easy way to install there as well.
+
+### Other OSes
+
+On other Operating Systems, please follow these steps:
+
+1. Download the latest release for your operating system from the [releases] page.
+1. Unpack the archive to any directory.
+1. Run the `./slackdump` or `slackdump.exe` executable (see note below).
+
+[releases]: https://github.com/rusq/slackdump/releases/
+
+> [!NOTE]
+> On Windows and macOS you may be presented with "Unknown developer" window.
+> The reason for this is that the executable hasn't been signed by
+> the developer certificate.
+
+  To work around this:
+
+  - **on Windows**: click "more information", and press "Run
+    Anyway" button.
+  - **on macOS** 14 Sonoma and prior:  open the folder in Finder, hold Option
+    and double click the executable, choose Run.
+  - **on macOS** 15 Sequoia and later:  start the slackdump, OS will show the
+    "Unknown developer" window, then go to System Preferences -> Security and
+    Privacy -> General, and press "Open Anyway" button.
+
+## Basic Slackdump Navigation
+
+Most of the Slackdump CLI uses this basic navigation:
+
+* Use TAB to navigate between fields
+* Use up and down arrows to navigate between items on a list
+* Use ENTER to select an option
+* Use ESC to go back to a higher-level menu
+
+## Step by Step Walkthrough
+
+Use the wizard to get started:
+
+![screenshot of starting the wizard](runwizard.jpeg)
+
+Navigate the menus and select "Export" (not "Dump"). You will be asked for a login.
```

Comment:
The login instructions in `Export` are different from what is shown below.

We need to add a section on Workspace Login via `Workspace > New`. The instructions below are for that approach.